libs/datamodel/connectors/datamodel-connector/src/lib.rs

```diff
@@ -48,6 +60,9 @@ pub enum ConnectorCapability {
     MultipleIndexesWithSameName,
     Enums,
     Json,
+    AutoIncrementAllowedOnNonId,
```

Comment:
Did the plan we discussed about going with a single enum variant containing booleans not work out?

Reply:
That one did not work with our logic for combined connectors.